### PR TITLE
Added edit session feature to load BatchConnect form with existing session parameters

### PR DESCRIPTION
**Pull request by abujeda**
An idea for a new feature:
Add an edit button into the session card to pre-load the Interactive Apps form with the session launch parameters.

This might be useful if users need to adjust few parameters when launching a new session

### CODE REVIEW

**Comment by abujeda** [comment]
demo:
edit_session

**Comment by abujeda** [comment]
I'll wait until #3352 is completed and will give it another go

**Comment by johrstrom** [comment]
#3352 has been merged so you can rebase now.

File: apps/dashboard/app/views/batch_connect/sessions/card/_card_header.html.erb [hunk 1]
```diff
@@ -20,6 +20,8 @@
       <span class="card-text"> | </span>
       <%- end -%>
       <%= status(session) %>
+      <span class="card-text"> | </span>
+      <%= edit(session) %>
```
[the remark this answers is not in the document]
Comment (abujeda) [reply apps/dashboard/app/views/batch_connect/sessions/card/_card_header.html.erb:24]: I didn't think of that. Will update to show only on completed sessions

**Pull request merged**